src/main.rs

```diff
@@ -265,15 +276,19 @@ fn main() -> Result<()> {
                         let kstr = utils::convert_to_string(ksym)
                             .context("Couldn't convert ksym to string")?;
 
-                        sequence.push(kstr.to_owned());
-
                         if app_config.hint_chars.contains(kstr) {
                             info!("Adding '{}' to key sequence", kstr);
                             pressed_keys.push_str(kstr);
+                        } else if window_cmd_map.contains_key(kstr) {
+                            window_command = window_cmd_map.get(kstr).unwrap();
+                            info!("Changing window_command to '{window_command:?}'");
+                            continue;
```

Comment:
Wouldn't this cause the sequence `fxf` to change the command to `Kill` in the middle of a focus sequence if there's a window with the label `fa`? Maybe I'm misunderstanding the code but I think we should require to the special code to be at the beginning of the sequence in order to be considered.

Reply:
Yeah, I did just side step most of the other logic, I'll see how well I can get them to play together...